fdbcli/DebugCommands.actor.cpp

```diff
@@ -279,21 +279,20 @@ ACTOR Future<bool> checkallCommandActor(Database cx, std::vector<StringRef> toke
 	state bool checkAll = false; // do not return on first error, continue checking all keys
 	state KeyRange toCheck = backupLogKeys;
 
-	if (tokens.size() == 2 && tokens[1] == "ALL"_sr) {
```

Comment:
do we want to retain this `checkall ALL` setting?

Reply:
Probably not.

Reply:
checkall ALL runs checking on backupLogKeys, which is supported by the new input.